.github/workflows/pw_aws_ci.yaml

```
@@ -97,8 +99,8 @@ jobs:
       with:
         path: ${{ github.run_id }}/HOMEgfs
         submodules: 'recursive'
-        repository: ${{ needs.fetch-branch.outputs.repo }}
-        ref: ${{ needs.fetch-branch.outputs.branch }}
+        repository: ${{ steps.git-branch.outputs.BRANCH }}
+        ref: ${{ steps.git-branch.outputs.REPO }}
```

Is this supposed to be

```suggestion
        repository: ${{ steps.fetch-branch.outputs.BRANCH }}
        ref: ${{ steps.fetch-branch.outputs.REPO }}
```